scripts/predictor/README.md

```diff
@@ -0,0 +1,59 @@
+# WHAT IS THIS SCRIPT SET FOR?
+
+This set of scripts is intended to help devs to find out if the contracts of a given commit / branch in a repo will reproduce exactly the contracts deployed in the blockchain.
```

Comment:
This set of scripts is intended to help devs to find out if the contracts ABI of a given commit / branch in a repo corresponds to the contracts deployed in the blockchain.